frappe/custom/doctype/customize_form/customize_form.js

```diff
@@ -89,7 +89,7 @@ frappe.ui.form.on("Customize Form", {
 
 	setup_sortable: function (frm) {
 		frm.doc.fields.forEach(function (f) {
-			if (!f.is_custom_field) {
+			if (!f.is_custom_field || f.is_system_generated) {
```

Comment:
This has been added temporarily to make system generated fields unsortable, since we aren't creating a property setter for `insert_after` as of now.

Reply:
Feature to rearrange standard and system generated fields being worked on separately https://github.com/frappe/frappe/pull/19822